tests/sync/test_chat_type_detection.py

```
@@ -0,0 +1,226 @@
+        trial_chat = OngoingChat.objects.create(
+            session_key="str_trial_sess_1234",
+            chat_type=ChatType.TRIAL_PROFESSIONAL,
+            is_patient=False,
+            chat_history=[],
+            summary_for_next_call=[],
+        )
+        self.assertIn("trial professional", str(trial_chat).lower())
+
+        # Professional
+        pro_user = User.objects.create_user(
+            username="str_pro", password="testpass", email="str_pro@example.com"
```

Ruff (0.15.5)

[error] Possible hardcoded password assigned to argument: "password"

(S106)